src/repositories/CancelableResponse.ts

```diff
@@ -54,3 +54,9 @@ export class CancelableResponse<Data> {
 const noop = () => {
     return;
 };
+
+export type Canceler = (message?: string) => void;
```

Comment:
Export ``Canceler`` type so that we do not rely on ``Axios`` anymore on the app side